tests/cli_tests.py

```diff
@@ -57,3 +60,41 @@ def test_import_controlled_list(self):
         self.assertTrue(List.objects.filter(pk=list_pk).exists())
 
     ### TODO Add test for creating new language if language code not in db but found in import file
+
+
+class RDMToControlledListsETLTests(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+
+        skos = SKOSReader()
+        rdf = skos.read_file(
+            os.path.join(PROJECT_TEST_ROOT, "data/concept_label_test_collection.xml")
+        )
+        ret = skos.save_concepts_from_skos(rdf)
+
+        client = Client()
+        client.login(username="admin", password="admin")
+        response = client.get(
+            reverse(
+                "make_collection",
+                kwargs={"conceptid": "7c90899a-dbe9-4574-9175-e69481a80b3c"},
+            )
+        )
+
+    def test_migrate_collections_to_controlled_lists(self):
+        output = io.StringIO()
+        management.call_command(
+            "controlled_lists",
+            operation="migrate_collections_to_controlled_lists",
+            collections_to_migrate=["Concept Label Import Test"],
+            host="http://localhost:8000/plugins/controlled-list-manager/item/",
+            preferred_sort_language="en",
+            overwrite=False,
+            stdout=output,
+        )
+
+        self.assertTrue(List.objects.filter(name="Concept Label Import Test").exists())
+        self.assertTrue(
+            ListItem.objects.filter(id="89ff530a-f350-44f0-ac88-bdd8904eb57e").exists()
+        )
```

Comment:
Asserting against a UUID is not as informative as something like this:

```suggestion
        imported_list = List.objects.get(name="Concept Label Import Test")
        imported_items = imported_list.list_items.all()
        self.assertEqual(len(imported_items), 3)
```